lib/ansible/modules/files/template.py

```diff
@@ -30,13 +30,14 @@
        (U(http://jinja.pocoo.org/docs/)) - documentation on the template
        formatting can be found in the Template Designer Documentation
        (U(http://jinja.pocoo.org/docs/templates/)).
-     - "Six additional variables can be used in templates:
+     - "Seven additional variables can be used in templates:
        C(ansible_managed) (configurable via the C(defaults) section of C(ansible.cfg)) contains a string which can be used to
```

Comment:
I wouldn't keep a count of how many additional variables can be used. Just list them !